_includes/pagination_buttons.html

```diff
@@ -0,0 +1,30 @@
+<div class="d-flex justify-content-center">
```

Comment:
Pagination looks like it's working great but the "Next/Previous" buttons appear even if there isn't a next or previous to go to

Reply:
Right. That's how we've had it set up on the blog page: https://pytorch.org/blog/. Do you want me to remove the disabled button from here and from the blog?

Reply:
Oh I didn't realize that was an existing pattern. If that's how it displays already then probably best to leave it